clustered/client/src/main/java/org/ehcache/clustered/client/config/ClusteringServiceConfiguration.java

```diff
@@ -144,7 +144,7 @@ public ClusteringServiceConfiguration(URI clusterUri, Timeouts timeouts, boolean
     this.autoCreate = autoCreate;
     this.serverConfiguration = serverConfig;
     this.timeouts = Objects.requireNonNull(timeouts, "Operation timeouts cannot be null");
-    this.properties = Objects.requireNonNull(properties, "Properties cannot be null");
+    this.properties = cloneProperties(Objects.requireNonNull(properties, "Properties cannot be null"));
```

Comment:
You could do `(Properties) Objects.requireNonNull(properties, "Properties cannot be null").clone();` and be done with it.